src/main/java/StringManipulation.java

```diff
@@ -1,33 +1,94 @@
 public class StringManipulation implements StringManipulationInterface {
 
```

Comment:
Modify or add additional test cases in StringManipulationTest.java. If new test cases failed, modify this file so that it passes the said test cases.